scope_shared/scope/testing/test_database/test_create_scheduled_activities.py

```diff
@@ -0,0 +1,84 @@
+import datetime
+from typing import Callable, List
+
+import scope.database.collection_utils as collection_utils
+import scope.database.date_utils as date_utils
+import scope.database.patient.activities
+import scope.database.patient.scheduled_activities
+import scope.schema
+import scope.schema_utils as schema_utils
+
+
+def _create_scheduled_activities_assertions(scheduled_activities: List[dict]) -> None:
+    schema_utils.assert_schema(
+        data=scheduled_activities,
+        schema=scope.schema.scheduled_activities_schema,
+    )
+    for scheduled_activity_current in scheduled_activities:
+        assert not scheduled_activity_current["completed"]
+        assert scheduled_activity_current["dueType"] == "Exact"
+
+
+# TODO: Discuss w/ James if this can parametrized.
+def test_create_scheduled_activities(
```

Comment:
split this into multiple tests, so that understanding any one test doesn't require understanding any state that might be accidentally or intentionally carried between tests